### PR TITLE
Add build notes to the README.md file for 3.0.0 branch

### PR DESCRIPTION
**Pull request by Akila94**
## Add build notes to the README.md file for 3.0.0 branch

**Doc Issue:** N/A

**Applicable Labels:** OB3

------

### Development Checklist

1. [ ] Built complete solution with pull request in place.
2. [ ] Ran checkstyle plugin with pull request in place.
3. [ ] Ran Findbugs plugin with pull request in place.
4. [ ] Ran FindSecurityBugs plugin and verified report.
5. [ ] Formatted code according to WSO2 code style.
6. [x] Have you verify the PR does't commit any keys, passwords, tokens, usernames, or other secrets?
7. [ ] Migration scripts written (if applicable).
8. [x] Have you followed secure coding standards in [WSO2 Secure Engineering Guidelines](http://wso2.com/technical-reports/wso2-secure-engineering-guidelines)?

### Testing Checklist

1. [ ] Written unit tests.
2. [ ] Documented test scenarios(link available in guides).
3. [ ] Written automation tests (link available in guides).
4. [ ] Verified tests in multiple database environments (if applicable).
5. [ ] Verified tests in multiple deployed specifications (if applicable).
6. [ ] Tested with OBBI enabled  (if applicable).
7. [ ] Tested with specification regulatory conformance suites  (if applicable).

**Automation Test Details**

| Test Suite        | Test Script IDs   |
| ----------------- | ----------------- |
| Integration Suite | *TCXXXXX, TCXXXX* |

**Conformance Tests Details**

| Test Suite Name  | Test Suite Version | Scenarios  | Result   |
| ---------------- | ------------------ | ---------- | -------- |
| *Security Suite* | *VX.X*             | *Foo, Bar* | *Passed* |

## Resources

**Knowledge Base:** https://sites.google.com/wso2.com/open-banking/

**Guides:** https://sites.google.com/wso2.com/open-banking/developer-guides

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied. See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+
+# WSO2 Financial Services Accelerator
```
Comment (Ashi1993) [inline README.md:19]: Should be Open Banking Accelerator

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
...
+Financial Services Accelerator a collection of technologies that increases the speed and reduces the complexity of adopting open banking compliance.
+
+### Building from the source
+
+If you want to build Financial Services Accelerator from the source code:
+
+1. Install Java8 or above.
+1. Install [Apache Maven 3.0.5](https://maven.apache.org/download.cgi) or above.
+1. Install [MySQL](https://dev.mysql.com/doc/refman/5.5/en/windows-installation.html).
+1. To get the Financial Services Accelerator from [this repository](https://github.com/wso2/financial-services-accelerator.git), click **Clone or download**.
```
Comment (Ashi1993) [inline README.md:30]: lets mention to select he relevant branch and then download

Comment (Akila94) [reply README.md]: Fixed with https://github.com/wso2/financial-services-accelerator/pull/221/commits/e755ebde386e6560d9acf08b41c55992ca7d6562

File: README.md (hunk 1 above)
Comment (Akila94) [reply README.md]: Fixed with https://github.com/wso2/financial-services-accelerator/pull/221/commits/e755ebde386e6560d9acf08b41c55992ca7d6562

File: README.md [hunk 3]
```diff
@@ -1 +1,68 @@
...
+   | ```mvn clean install -P solution``` | This cleans the folders and starts building the solution pack from scratch. Finally creates the accelerator zip files containing the artifacts required to setup the deployment. |
+
+1. Once the build is created, navigate to the relevant folder to get the accelerator for each product.
+
+| Product                                 | Toolkit Path                                            |
+      |:----------------------------------------|:--------------------------------------------------------|
+| ```Identity Server Accelerator```       | `/open-banking-accelerator/accelerators/ob-is/target`   |
+| ```API Manager Accelerator```           | `/open-banking-accelerator/accelerators/ob-apim/target` |
+| ```Business Intelligence Accelerator``` | `/open-banking-accelerator/accelerators/ob-bi/target`   |
+
```
Comment (lalaji) [inline README.md:49]: shall we link,documentation link to install accelerators here?

File: README.md [hunk 4]
```diff
@@ -1 +1,68 @@
...
+Open Banking Accelerator a collection of technologies that increases the speed and reduces the complexity of adopting open banking compliance.
+
+### Building from the source
+
+If you want to build Open Banking Accelerator from the source code:
+
+1. Install Java8 or above.
+1. Install [Apache Maven 3.0.5](https://maven.apache.org/download.cgi) or above.
+1. Install [MySQL](https://dev.mysql.com/doc/refman/5.5/en/windows-installation.html).
```
Comment (lalaji) [inline README.md:29]: mysql doesn't need to be install on build time. Isn't it? It need for runtime as. I know.

File: README.md [hunk 5]
```diff
@@ -1 +1,68 @@
...
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied. See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+
+# WSO2 Open Banking Accelerator
+
+Open Banking Accelerator a collection of technologies that increases the speed and reduces the complexity of adopting open banking compliance.
```
Comment (lalaji) [inline README.md:21]: Let's add documentation link for introductory page,after your sentence for more info.

File: README.md [hunk 6]
```diff
@@ -1 +1,68 @@
...
+1. Once the build is created, navigate to the relevant folder to get the accelerator for each product.
+
+| Product                                 | Toolkit Path                                            |
+      |:----------------------------------------|:--------------------------------------------------------|
+| ```Identity Server Accelerator```       | `/open-banking-accelerator/accelerators/ob-is/target`   |
+| ```API Manager Accelerator```           | `/open-banking-accelerator/accelerators/ob-apim/target` |
+| ```Business Intelligence Accelerator``` | `/open-banking-accelerator/accelerators/ob-bi/target`   |
+
+
+### Running the products
```
Comment (lalaji) [inline README.md:51]: Let's say install & running OB accelerators

File: README.md [hunk 7]
```diff
@@ -1 +1,68 @@
...
+
+Please refer the following READ.ME files to run the products.
+
+| Product                            | Instructions Path                         |
+|:-----------------------------------|:------------------------------------------|
+| ```Identity Server```              | `/wso2-obiam-accelerator-3.0.0/README.md` |
```
Comment (lalaji) [inline README.md:57]: shall we use ob accelerator names..coz at the end,we r configuring accelerators on base products

File: README.md (hunk 6 above)
Comment (Akila94) [reply README.md]: Fixed with https://github.com/wso2/financial-services-accelerator/pull/221/commits/66b701950d67c10aeb2be51197e5a65f9031b575

File: README.md (hunk 5 above)
Comment (Akila94) [reply README.md]: Fixed with https://github.com/wso2/financial-services-accelerator/pull/221/commits/66b701950d67c10aeb2be51197e5a65f9031b575

File: README.md (hunk 4 above)
Comment (Akila94) [reply README.md]: Fixed with https://github.com/wso2/financial-services-accelerator/pull/221/commits/66b701950d67c10aeb2be51197e5a65f9031b575

**Review by lalaji — APPROVED** [review]

**Pull request merged**